v2/cli/init_templates_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/stretchr/testify/require"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func TestExecTemplate(t *testing.T) {
+	bytes, err := execTemplate(
+		[]byte(`hello {{.ID}}`),
+		struct{ ID string }{ID: "world"},
+	)
+
+	require.NoError(t, err)
+
+	require.Equal(t, []byte(`hello world`), bytes)
+}
+
+func TestProjectFileTemplate(t *testing.T) {
+	testCases := []struct {
+		name        string
+		projectID   string
+		language    string
+		gitCloneURL string
+	}{
+		{"nonGit", "hello", "ts", ""},
+		{"git", "hello", "ts",
```

Comment:
Reminder not to initialize things without being explicit about what values go in which fields. That will be more resilient to change.